appveyor.yml

```diff
@@ -173,4 +182,9 @@ on_success:
 
 
 on_finish:
-  - ps: 'if (Test-Path ".\junit-results.xml") { (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\junit-results.xml)) }'
+  - ps: |
+      (new-object net.webclient).UploadFile(
```

Comment:
It is my personal opinion that if this fails for whatever reason, the build should not indicate as failing, because it has no bearing on whether the changes made to pyinstaller are good or bad.